app/views/home/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+<header>
+  <title>HackerNews</title>
+</header>
+
+<div class="page-header">
+  <%= link_to 'Hacker News', root_path, class: 'page-title' %> |
+  <%= link_to 'new', root_path, class: 'link' %> |
+  <%= link_to 'comments', root_path, class: 'link' %> |
+  <%= link_to 'submit', root_path, class: 'link' %>
+
+  <% if session[:user_id].present? %>
```

Comment:
I think you might have already created a `current_user` helper method inside your `ApplicationController`.  If you haven't already, I'd add that in and use that instead of referring to `session` inside your views.

So this (and similar places) would look like:
```
<% if current_user.present? %>
```